joshu.hpp

```diff
@@ -222,14 +369,14 @@ class imod_t {
 public:
   imod_t() = default;
   imod_t(imod_t const&) = default;
-  imod_t(imod_t &&) = default;
+  imod_t(imod_t&&) = default;
```

Comment:
It is recommended to remove the changes that are **unrelated to the explicit purpose** for the moment.